python/ray/data/tests/test_map.py

```diff
@@ -374,7 +374,7 @@ def test_map_batches_basic(ray_start_regular_shared, tmp_path, restore_data_cont
 
 def test_map_batches_extra_args(shutdown_only, tmp_path):
     ray.shutdown()
-    ray.init(num_cpus=2)
+    ray.init(num_cpus=3)
```

Comment:
is this needed as a result of code changes in this PR?

Reply:
This is because we are no longer fusing actors.